connect/src/__tests__/manage3IDs.test.ts

```diff
@@ -55,6 +58,22 @@ describe('3ID Management', () => {
 
   test('creates/loads did from in memory', async () => {})
 
+  //TODO This test fails, if you log the authclient and authserver you can see it returns
```

Comment:
@PaulLeCam mainly the test I wanted to try, and couldnt get to work. Also start using elsewhere but more or less just wrote everything and didnt actually step through to make sure things are working yet, getting there tmrrw still. just sharing for context, no need to prio or anything